File: aspnet-core/src/group5/Group5.SE347.L11_HelloWork.Application/Services/StateApplications/Dto/StateApplicationDto.cs
using SE347.L11_HelloWork.Entities;
using System;
using System.Collections.Generic;
using System.Text;
using Abp.AutoMapper;
using System.ComponentModel.DataAnnotations;
using Abp.Application.Services.Dto;

namespace Group5.SE347.L11_HelloWork.Application.Services.StateApplications.Dto
{
    [AutoMapFrom(typeof(StateApplication))]
    public class StateApplicationDto : EntityDto<int>
    {
        public int IDJobSeeker { get; set; }
        public int IDRecruitment { get; set; }
        public string State { get; set; }
    }
}

File: reactjs/src/app/groups/group5/services/dto/stateApplicationDto/stateApplicationDto.ts
export interface CreateStateApplicationInput {
    IDJobSeeker: number,
    IDRecruitment: number,
    State: string,
}

export interface UpdateStateApplicationInput extends CreateStateApplicationInput {
    id: number
}

export interface GetStateApplicationByJSInput {
    IDJobSeeker: number,
    IDRecruitment: number,
}